nginx.conf

```diff
@@ -94,6 +94,8 @@
     # Grafana
     rewrite ^/$ $scheme://$http_host/graph/;
     rewrite ^/graph$ /graph/;
+    rewrite ^/(user/password/send-reset-email)$ $scheme://$http_host/graph/$1;
+    rewrite ^/(login)$ $scheme://$http_host/graph/$1;
```

Comment:
Please add a comment explaining why it is needed. Is it a Grafana bug? If so, please add a link to their issue.

And why add those lines there, between `rewrite ^/graph$ /graph/;` and `location /graph` that clearly paired together?

Reply:
I have moved lines to a separate section and added the link.

Reply:
@atymchuk IMHO the current way is much cleaner than additional sections with equal comments.